### ISSUE TITLE
Data set problem consultation

### ISSUE THREAD

**Issue by yihp** [leo726/MedVisioChat#1]
Are all data sets included in the data download link of the README file?

**Comment by leo726** [comment]
Yes, all datasets have been collected, organized, and included in the link.
